apps/backend/src/services/container_service.py

```
@@ -2,34 +2,32 @@
 Service layer for container-related business logic.
 """
 
-import asyncio
+from datetime import UTC, datetime
 import json
 import logging
-from datetime import datetime, timezone
-from typing import List, Optional, Dict, Any
+
+Optional
```

Ruff (0.12.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)